charts/prometheus-nginx-exporter/templates/service.yaml

```
@@ -12,6 +12,9 @@ metadata:
     {{- include "prometheus-nginx-exporter.labels" . | indent 4 }}
 spec:
   type: {{ .Values.service.type }}
+  {{- if ne .Values.service.externalTrafficPolicy "Cluster" }}
```

Can we please let k8s set the default giving the user an option to override that default?

```suggestion
  {{- if .Values.service.externalTrafficPolicy }}
```